### DISCUSSION TITLE
Active Filters

### DISCUSSION THREAD

**Discussion by yagogak** [malzariey/filament-daterangepicker-filter#40] [Ideas]
Hi @malzariey 

The filter works pretty well, but it doesn't register itself as an active filter in the "active filters" bar , and there is no way to disable selection.
Did i miss something ?

**Comment by malzariey** [comment]
Using withIndicator() will include the filter in the active filters count. As for now you can reset the value by resetting all filters, I am working on clearing the filter on the next coming release soon.
